management/virtualmachine/client.go

```diff
@@ -665,7 +665,7 @@ func (self VirtualMachineClient) createSshConfig(certPath, userName string) (SSH
 	publicKey.Fingerprint = fingerprint
 	publicKey.Path = "/home/" + userName + "/.ssh/authorized_keys"
```

Comment:
do you think we can just accept certPath as input to this method? hardcoded home path wouldn't work on windows or mac.